src/Features/Core/Portable/Completion/CompletionHelper.cs

```diff
@@ -81,46 +81,9 @@ public bool MatchesPattern(CompletionItem item, string pattern, CultureInfo cult
                 matchedAdditionalFilterText);
         }
 
-        private PatternMatch? GetMatch(
-            string text,
-            string pattern,
-            bool includeMatchSpans,
-            CultureInfo culture)
-        {
-            // If the item has a dot in it (i.e. for something like enum completion), then attempt
-            // to match what the user wrote against the last portion of the name.  That way if they
-            // write "Bl" and we have "Blub" and "Color.Black", we'll consider the latter to be a
-            // better match as they'll both be prefix matches, and the latter will have a higher
-            // priority.
-
-            var lastDotIndex = text.LastIndexOf('.');
```

Comment:
Deleted the special logic here, enum providers now use AdditionalFilterText to achieve this